apps/admin/frontend/src/screens/manual_data_import_precinct_screen.tsx

```diff
@@ -175,6 +180,12 @@ interface TempManualTally {
   readonly numberOfBallotsCounted: number;
 }
 
+interface TempWriteInCandidate {
+  readonly id: string;
+  readonly name: string;
+  readonly contestId: string;
+}
+
```

Comment:
The idea here is that you don't actually create new write-in candidates until you save the tally.